src/linear_algebra/finite_dimensional.lean

```diff
@@ -1141,8 +1141,20 @@ section span
 
 open submodule
 
+variable (K)
+
+protected noncomputable def set.finrank (s: set V) : ℕ := finrank K (span K s)
+
+variable {K}
+
+lemma set.finrank_mono (s t: set V) [finite_dimensional K V] : s ⊆ t → s.finrank K ≤ t.finrank K :=
+begin
+  intro h,
+  exact finrank_mono (span_mono h),
+end
```

Comment:
May as well move `h` before the colon to avoid `intro`
```suggestion
lemma set.finrank_mono [finite_dimensional K V] {s t : set V} (h : s ⊆ t) : s.finrank K ≤ t.finrank K :=
finrank_mono (span_mono h)
```
I've also make `s t` implicit, since they're implied by `h`

Reply:
Thank you!